### DISCUSSION TITLE
Auditing/Sessions + maintenance

### DISCUSSION THREAD

**Discussion by maekee** [netbox-community/netbox#7638] [Q&A]
Hello,
Are there any session/auditing/logging information about which users are using NetBox. It would be nice if we could see which users are logged in and what they do. Yes, i know that we can look in the change log for changes, but just who have logged on and when would be valuable.

I know that i can activate for example Kerberos in apache and see the users there in our environment, but would be nice to see this in netbox somehow.

Are this maintenance task running when installing NetBox or is it recommended to install manually?
https://netbox.readthedocs.io/en/stable/administration/housekeeping/

_"NetBox includes a housekeeping management command that should be run nightly."_
Should as in should be activated by the admin or should as in this should run automatically when installing netbox :)

**Comment by candlerb** [comment]
> Should as in should be activated by the admin or should as in this should run automatically when installing netbox :)

The former.  It also says:

"This script can be linked from your cron scheduler's daily jobs directory (e.g. /etc/cron.daily)"

That's the suggested way for the admin to activate it to be run nightly.  Installing netbox doesn't do this for you, as you might have some other way you prefer to run and monitor scheduled tasks.
